nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/assets/styles/_app.scss

```diff
@@ -165,6 +170,10 @@
     .mat-mdc-paginator {
         background-color: unset !important;
     }
+
+    .mdc-dialog__content {
```

Comment:
So either we want to move the font-size rule under the `.dialog-tab-content` 

```suggestion
    .dialog-tab-content,
    .dialog-content {
        height: 50vh;
        
        .mdc-dialog__content {
             font-size: 14px;
        }
    }

```

and then we don't need to use `!important` or if we want to force all dialog content text to be `font-size:14px` then we can keep it as but please remove the `font-size: 14px` rule from the other dialog style sheets where we are doing things like:

```
.mdc-dialog__content {
        padding: 0 16px;
        font-size: 14px;

.
.
.
    }
```

It looks like there are only five other such dialogs: 
![Screenshot 2024-05-09 at 9 52 22 AM](https://github.com/apache/nifi/assets/6797571/0aca96bd-fdd6-4f0a-b838-25cbd5476027)